doc/lua_api.txt

```diff
@@ -6297,9 +6299,11 @@ Misc.
     * `method` is a string identifying the compression method to be used.
     * Supported compression methods:
         * Deflate (zlib): `"deflate"`
+        * ZStd: `"zstd"`
     * `...` indicates method-specific arguments. Currently defined arguments
       are:
         * Deflate: `level` - Compression level, `0`-`9` or `nil`.
+        * ZStd: `level` - Compression level. `3` is the default.
```

Comment:
range?

Reply:
The range appears to be able to change dependent on the version of ZSTD.
If I'm reading `zstd.h` correctly, `0-3` is a safe range across all versions. Outside of that, no idea.

Reply:
Huh, guess we're leaving that undocumented then.

Reply:
Any downsides clamping to the supported range? What will ZStd do if a value out of range is provided?